http/src/main/resources/reference.conf

```diff
@@ -167,10 +167,26 @@ groups {
 
 gke {
   cluster {
-    location = "us-central1-a"
+    location = "us-central1-a",
+    region = "us-central1",
+    authorizedNetworks = ["69.173.127.0/25",
+                          "69.173.124.0/23",
+                          "69.173.126.0/24",
+                          "69.173.127.230/31",
+                          "69.173.64.0/19",
+                          "69.173.127.224/30",
+                          "69.173.127.192/27",
+                          "69.173.120.0/22",
+                          "69.173.127.228/32",
+                          "69.173.127.232/29",
+                          "69.173.127.128/26",
+                          "69.173.96.0/20",
+                          "69.173.127.240/28",
+                          "69.173.112.0/21"
+    ]
   }
   defaultNodepool {
-    machineType = "g1-small"
+    machineType = "n1-standard-1"
```

Comment:
testing shows we need at least .7 cpu as opposed to the .5 g1-small has for the default nodepool system pods. Bumping this machine size slightly